### PR TITLE
Kartikey_Assignment_27_35min

### PR DESCRIPTION
**Pull request by KartikeyDJune20**
image

### CODE REVIEW

File: src/kartikeyDJune20/MeetingRoom.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+			System.out.println("Takshila meeting room is avaibale");
+	}
+
+	public static void main(String[] args) {
+		MeetingRoom meetingRoom = new MeetingRoom();
+		meetingRoom.bookMeetingRoom();
+		meetingRoom.bookmeetigRoom("Takshila", 3);
+		meetingRoom.bookmeetigRoom("agrafort");
+		meetingRoom.displayInfo();
+		meetingRoom.availableMeetingRoom();
```
Comment (harshvegada) [inline src/kartikeyDJune20/MeetingRoom.java:85]: hi @KartikeyDJune20 
here displayInfo() method showing wrong information, while i m trying to book available meeting room,
after calling displayInfor() method call bookmeetingRoon("PratapGadh");
here logic is failing  little hint:

what you should do is define String array named as meetingRoomName and second array which contains hours.
when someone booked room that time you should update the same index value.

String[] meetingRoomName = {"Nalanda", "Takshashila", "AgraFort","PratapGadh"};
int hours[] = new int[4];

this hint may help you to think on this. for now we know array which can store multiple value, so think on this

Comment (KartikeyDJune20) [reply src/kartikeyDJune20/MeetingRoom.java:85]: Sure i will incorporate the changes today

File: src/kartikeyDJune20/MeetingRoomUsingEnum.java [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+				System.out.println(roomName+" room you want is already booked for "+agrafortBookingTime+" hrs");
+			break;
+		case PRATAPGARDH:
+			if (pratapGardhBooked == false) {
+				pratapGardhBooked = true;
+				pratapGrardhBookingTime = time;
+			} else
+				System.out.println(roomName+" room you want is already booked for "+pratapGrardhBookingTime+" hrs");
+			break;
+		default:
```
Comment (KartikeyDJune20) [inline src/kartikeyDJune20/MeetingRoomUsingEnum.java:49]: Please ignore this as i forget it to comment .. we are using enum class so there is no use of it..

**Review by harshvegada — COMMENTED** [review]
i guess this is duplicated class of PR 866

**Pull request merged**